ISISComputingGroup/EPICS-ioc
# Implement VTI SPC Macros

### Description of work

Added macros allowing the user to configure the VTI software pressure control designed for EMU. 

### To test

This isn't easy to test manully, but one of the easiest ways to see it working is to run the test suite with it open in the GUI. The test suite also includes all the macros which can be edited to see if the GUI is updating properly and acting as it should. 

### Acceptance criteria

Updating the macros in the GUI allows the VTI tab to be accessed and the correct values are set. 

Partially Resolves: ISISComputingGroup/IBEX#6947

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
